source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/height-updater.js

```diff
@@ -0,0 +1,8 @@
+﻿(function ($, ko) {
+    ko.bindingHandlers.heightUpdater = {
+        update: function (element) {
+            $(element).height(1);
```

Comment:
As we saw, scrollHeight is at least equals than height. To obtain an accurate value, we have to set height to 1